client/src/components/RequestHero.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
```

Comment:
RequestHero? What does that mean?

Reply:
A hero is like a page header that gives you context before the fold usually with one or two CTA's. I do think this is a bit of a stretch here - I was trying to avoid something like `RequestResource` though maybe something like `RequestResourceDetails` might be better.

Reply:
ok upated to `RequestResourceContact`